x-pack/test_serverless/functional/test_suites/observability/rules/rules_list.ts

```diff
@@ -0,0 +1,856 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from 'expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import {
+  createRule,
+  disableRule,
+  enableRule,
+  runRule,
+  createIndexConnector,
+  snoozeRule,
+} from '../../../../api_integration/test_suites/common/alerting/helpers/alerting_api_helper';
+
+export default ({ getPageObject, getService }: FtrProviderContext) => {
+  const svlCommonNavigation = getPageObject('svlCommonNavigation');
+  const svlTriggersActionsUI = getPageObject('svlTriggersActionsUI');
+  const svlObltNavigation = getService('svlObltNavigation');
+  const testSubjects = getService('testSubjects');
+  const supertest = getService('supertest');
+  const find = getService('find');
+  const retry = getService('retry');
+  const toasts = getService('toasts');
+
+  async function refreshRulesList() {
+    await svlCommonNavigation.sidenav.clickLink({ text: 'Alerts' });
+    await testSubjects.click('manageRulesPageButton');
+  }
+
+  async function failRule({
```

Comment:
@ymao1 @pmuellr @mikecote @XavierM 
Hi!
Please, help me with your thoughts.
AsI understood in the new serverless tests the plugin solution (as it is done in x-pack/test/functional_with_es_ssl) is something we want to avoid. We do not want to add a new rule types only for testing and neither rewriting an existing type.
I've talked with @ymao1 and she mentioned that I can probably fail a rule by running it at a very fast interval like 1s and repeatedly try disabling/enabling until you encounter an error. That's I implemented it in that function.
I've tested in using flakiness framework(https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/3180) and it seems working. 
But I still have doubts. What if with some conditions and in some environments it still be flaky? What do you think? is it good enough or maybe you have some other ideas?

Reply:
Ya, I'm not sure that's a good approach. Some day we'll get all this stuff so it doesn't fail in these conditions, and then this test will break! :-). Right? (hehe)

I think at some point we **ARE** going to need to add some custom plugins so we can add custom rules.  Which would make doing things like this throw away code.  Is there anything realy stopping us from doing that today?  We'd need to edit the config to add the appropriate plugin path (I wonder - would our existing plugins work???), and then 🤞🏻 hope the serverless Kibana will actually handle that (not clear to me it will).

Reply:
Yea, I was under the impression that we did not want to introduce the custom test rule types that we have for non serverless. In the `Kibana Serverless e2e Test Guide`, the first guidelines is `Serverless tests must be designed to pass in “local serverless” (PR CI) AND in serverless MKI projects.`. AFAIK in a serverless MKI project, we will not be able to have custom plugins with custom rules as I believe testing involves creating a serverless project in QA and pointing the functional tests to that project.

Reply:
If the goal is to fail the rule run, you could update the rule to contain parameters that will fail the rule run. This way we don't need to create a custom rule type and allows the test to run on serverless as well.

For example, if using the Elasticsearch Query rule, you can use the DSL to provide something that will cause the query, and in return, the rule to fail running.

Using DSL like this will cause the rule to fail on every rule run 100% of the time.
```
{
  "query": {
    "match_everything": {}
  }
}
```

Reply:
@mikecote we do not have Elasticsearch Query rule type in Observability in Serverless.
Only these types:
![Uploading Screenshot 2023-09-20 at 13.20.11.png…]()

Reply:
@XavierM 's PR will add ES Query rule in O11y serverless soon => https://github.com/elastic/kibana/pull/166032 can it wait for that? Or can we find an O11y rule that would fail to run 100% of the time by giving it a parameter that would fail the execution?